core/backend/src/IModelExporter.ts

```diff
@@ -117,7 +117,7 @@ export abstract class IModelExportHandler {
    * @param schema The schema to export
    * @note This should be overridden to actually do the export.
    */
-  protected onExportSchema(_schema: Schema): void { }
+  protected async onExportSchema(_schema: Schema): Promise<void> { }

```

Comment:
@scsewall I initially tried to keep this from being async, but the code was much worse. I can imagine paint points unless most of these handlers become async, thoughts?

Reply:
Most of the `onExport*` callbacks are handling a single entity, so I am not sure that making them async is required. However, I can see the merit in making `onExportSchema` async.